### ISSUE TITLE
[REQ] Steam Overlay Support

### ISSUE THREAD

**Issue by MrMurphyMGSV** [pizzaboxer/bloxstrap#2173]
### Acknowledgement of preliminary instructions

- [X] I have read and acknowledged the preliminary instructions.

### What idea do you have?

Adding Roblox as a non-steam game won't show the overlay, so i wanted to see if it was a good idea to add Steam Overlay to Bloxstrap so i can shift+tab and talk to my friends while keeping roblox opened up instead of alt tabbing.

**Comment by pizzaboxer** [comment]
I doubt this would be possible for the same reason that Discord Overlay doesn't work either. I'll take a look and see if it's possible.

**Comment by bluepilledgreat** [comment]
This is highly likely impossible due to Hyperion.

**Comment by Psyden57** [comment]
It's been done before though
https://github.com/Corecii/roblox_steam_launcher
(Doesn't work right now because it has old regedit paths)

**Comment by DrakeisaCake** [comment]
> (Doesn't work right now because it has old regedit paths)

Is it just possible to update the paths?

**Comment by DolphinOfficial** [comment]
Can we come back to this with the new Steam Controller release?

**Comment by alec-catastrophic** [comment]
yeah i wanna use the steam controller as well even if it registers as an xbox controller

**Comment by snowboyz0825** [comment]
> Can we come back to this with the new Steam Controller release?

You are technically able to get the steam controller to run it using sisr, which makes any steam input emulate a 360 controller, but it's highly annoying
